### PR TITLE
Fixes logging related issue, updates template, small CSS extra

### PR DESCRIPTION
**Pull request by XinYenFon**

### CODE REVIEW

File: Themes/default/Agreement.template.php [hunk 1]
```diff
@@ -58,29 +58,29 @@ function template_main()
...
-		<div class="description">
+		<div class="information noup">
 			', sprintf($txt['privacy_policy_accepted'], timeformat($context['privacy_policy_accepted_date'], false)), '
 		</div>';
 		}
 
 		echo '
-		<div class="roundframe">
-			<div>', $context['policy'], '</div>
+		<div class="windowbg">
```
Comment (live627) [inline Themes/default/Agreement.template.php:74]: Without `noup` it looks like
Screenshot_2020-05-06 Privacy Policy

Comment (XinYenFon) [reply Themes/default/Agreement.template.php:74]: This should be correct behavior, not all windowbg / roundframe after -information- block required noup but I'll double check this with Gwen...

Comment (XinYenFon) [reply Themes/default/Agreement.template.php:74]: This should be correct behavior, not all windowbg / roundframe after -information- block required noup but I'll double check this with Gwen...

Comment (live627) [reply Themes/default/Agreement.template.php:74]: I just think it looks a little off as two separate blocks.

**Comment by MisterVector** [comment]
Thanks for helping out with this!

File: Themes/default/Agreement.template.php (hunk 1 above)
Comment (XinYenFon) [reply Themes/default/Agreement.template.php:74]: we can fix the small inconsistency, but as I said I'll double check this with other Front-End fellas and fix it (after sometime before Final release).

**Pull request merged**